docs/articles/data_storage.rst

```diff
@@ -256,6 +258,71 @@ Note that above the *force* parameter is necessary if running from a non-interac
 with a migration requires confirmation in the form of user prompt. If running programatically this parameter
 instead confirms your intention to proceed with the migration.
 
+Putting it all together
+  .. code-block::
+
+    import os
+    import synapseutils
+    import synapseclient
+
+    my_synapse_folder_to_migrate = "syn123"
+
+    external_bucket_name = "my-external-synapse-bucket"
+    external_bucket_base_key = "path/within/bucket/"
```

Comment:
Yes it can be - it would use the root of the bucket.

Though even with this passed in the migration does not adhere to this base_key. I still find files were placed into the root of the bucket.